core/tests/protractor_utils/ExplorationEditorPage.js

```diff
@@ -182,9 +193,13 @@ var ExplorationEditorPage = function() {
       explorationMetadataModalHeaderElement);
     await action.waitForAutosave();
 
-    await action.select(
-      'Exploration Language', expLanguageSelectorElement,
-      language);
+    await action.click(
+      'Exploration Language', expLanguageSelectorElementModal, true);
+
+    await action.click(
+      'Language input Choice takes too long to be visible. ',
+      languageChoiceOptionElement(language), true);
```

Comment:
Why are we clicking masked elements here?

Reply:
removed